### ISSUE TITLE
Import of PCAPKit crashes

### ISSUE THREAD

**Issue by huettenhain** [JarryShaw/PyPCAPKit#113]
In a clean virtual environment running Python 3.9.7, importing PyPCAPKit fails:
```
c:\Workspace\temporary>C:\Python39\python.exe -m venv venv

c:\Workspace\temporary>venv\Scripts\activate.bat

(venv) c:\Workspace\temporary>pip install pypcapkit[scapy]
Collecting pypcapkit[scapy]
  Using cached pypcapkit-0.16.1-py3-none-any.whl (406 kB)
Collecting aenum
  Using cached aenum-3.1.11-py3-none-any.whl (131 kB)
Collecting tbtrim>=0.2.1
  Using cached tbtrim-0.3.1-py2.py3-none-any.whl (4.4 kB)
Collecting chardet
  Using cached chardet-5.0.0-py3-none-any.whl (193 kB)
Collecting dictdumper~=0.8.0
  Using cached dictdumper-0.8.4.post2-py2.py3-none-any.whl (18 kB)
Collecting scapy
  Using cached scapy-2.4.5-py2.py3-none-any.whl
Installing collected packages: tbtrim, dictdumper, chardet, aenum, scapy, pypcapkit
Successfully installed aenum-3.1.11 chardet-5.0.0 dictdumper-0.8.4.post2 pypcapkit-0.16.1 scapy-2.4.5 tbtrim-0.3.1
WARNING: You are using pip version 21.2.3; however, version 22.2.1 is available.
You should consider upgrading via the 'c:\Workspace\temporary\venv\Scripts\python.exe -m pip install --upgrade pip' command.

(venv) c:\Workspace\temporary>python
Python 3.9.7 (tags/v3.9.7:1016ef3, Aug 30 2021, 20:19:38) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import pcapkit
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() got an unexpected keyword argument 'ABOR'
>>>
```

**System information**
A clear and concise description of your system information.
 - OS Version: Windows 10
 - Python Version: 3.9.7
 - Python Implementation: CPython

**Expected behavior**
Importing the module succeeds

**Comment by JarryShaw** [comment]
Bug fixed in version **0.16.2**.

**Comment by huettenhain** [comment]
Many thanks for the swift fix and release!
